### ISSUE TITLE
restore commas to cei:idno

### ISSUE THREAD

**Issue by NTsch** [icaruseu/mom-ca#1093]
Commas in `cei:idno` are currently removed by https://github.com/icaruseu/mom-ca/blob/master/my/XRX/src/mom/app/charter/xsl/cei2html.xsl#L1191. Keeping the comma is requested for the Fontenay collection, so this transformation should be removed unless a good reason for it emerges.

**Comment by larkvi** [comment]
Why is this desirable? What does a Fontenay msIdentifier look like?

**Comment by GVogeler** [comment]
`<idno>15 H 17, pièce 1</idno>`

**Comment by GVogeler** [comment]
I currently don't see a reason why the idno should obligatorily should not include a comma.

**Comment by GVogeler** [comment]
And if so this should be enforced by schema not by transformation for display, shouldn't it?

**Comment by larkvi** [comment]
Agreed, but from a modelling perspective, is piece 1 really part of its shelfmark, or is it just an item in a box which has a shelf mark? I think it is less than ideal, and was curious. Off the top of my head, something like nested `idno`s 
```
<idno>
  Box Number
  <idno>
    piece order (do we not have an existing element for ordering?)
  </idno>
</idno>
```
would seem like a better solution, but again, that is a modelling question and has nothing, as you say, to do with he transformation.
